include/tester/db/Database.hpp

```diff
@@ -82,4 +83,6 @@ class Database
     static constexpr auto DATABASE_NAME = "VoIP-Tester-DB";
     static constexpr auto MONGO = "mongodb://";
     static constexpr auto PORT = ":27017";
+
+    const std::string ID = "_id";
```

Comment:
rename to `ID_FIELD` and look that you've made this `const` which is a runtime constant instead of a `static constexpr` which is a compile-time constant.

Reply:
I can't do this `static constexpr` I tried.
If I do it `static constexpr` then `bsoncxx::builder::stream::document{} << ID << bsoncxx::oid{std::move(documentId)}` is invalid